src/js/dp/search.js

```diff
@@ -54,15 +54,19 @@ if (searchContainer) {
     } else {
       const fetchedDom = new DOMParser().parseFromString(responseText, "text/html");
 
-      const resultsCount = fetchedDom.querySelector(".search__summary__count").innerText;
-      const noResults = resultsCount.startsWith("0");
-      const noResultsMessage = document.querySelector('#results-zero')
-
-      if (noResults) {
-        if (noResultsMessage) noResultsMessage.classList.remove('hide');
-        searchContainer.querySelector("#results > ul").innerHTML = '';
-        searchContainer.querySelector(".search__pagination").innerHTML = '';
-        searchContainer.querySelector(".search__summary__count").innerText = '0';
+      let resultsCount = 0;
```

Comment:
This var is being initialised as an int but then is being passed a string without converting it